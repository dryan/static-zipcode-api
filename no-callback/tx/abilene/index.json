{"region": {"fips": "48", "name": "Texas", "abbr": "TX"}, "postal_codes": ["79698", "79699", "79697", "79607", "79606", "79605", "79604", "79603", "79602", "79601", "79608"], "name": "Abilene", "counties": [{"fips": "417", "name": "Shackelford"}, {"fips": "059", "name": "Callahan"}, {"fips": "253", "name": "Jones"}, {"fips": "441", "name": "Taylor"}]}