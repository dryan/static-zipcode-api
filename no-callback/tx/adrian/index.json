{"region": {"fips": "48", "name": "Texas", "abbr": "TX"}, "postal_codes": ["79001"], "name": "Adrian", "counties": [{"fips": "359", "name": "Oldham"}, {"fips": "117", "name": "Deaf Smith"}]}