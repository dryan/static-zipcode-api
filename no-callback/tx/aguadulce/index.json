{"region": {"fips": "48", "name": "Texas", "abbr": "TX"}, "postal_codes": ["78330"], "name": "Agua Dulce", "counties": [{"fips": "355", "name": "Nueces"}]}