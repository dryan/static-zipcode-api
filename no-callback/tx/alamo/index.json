{"region": {"fips": "48", "name": "Texas", "abbr": "TX"}, "postal_codes": ["78516"], "name": "Alamo", "counties": [{"fips": "215", "name": "Hidalgo"}]}