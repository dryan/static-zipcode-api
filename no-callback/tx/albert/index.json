{"region": {"fips": "48", "name": "Texas", "abbr": "TX"}, "postal_codes": ["78671"], "name": "Albert", "counties": [{"fips": "171", "name": "Gillespie"}]}