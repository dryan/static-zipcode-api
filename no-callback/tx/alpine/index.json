{"region": {"fips": "48", "name": "Texas", "abbr": "TX"}, "postal_codes": ["79830", "79831", "79832"], "name": "Alpine", "counties": [{"fips": "043", "name": "Brewster"}]}