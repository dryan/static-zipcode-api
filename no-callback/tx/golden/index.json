{"region": {"fips": "48", "name": "Texas", "abbr": "TX"}, "postal_codes": ["75444"], "name": "Golden", "counties": [{"fips": "499", "name": "Wood"}]}